src/sandbox_functions.cpp

```diff
@@ -1214,7 +1214,7 @@ Sandbox::BinaryInfo Sandbox::get_program_info_from_binary(const PackedByteArray
 
 		// Get all unmangled public functions from the guest program.
 		// Exclude functions that belong to the C/C++ runtime, as well as compiler-generated functions.
-		for (auto &function : machine.memory.all_unmangled_function_symbols()) {
+		for (std::string_view &function : machine.memory.all_unmangled_function_symbols()) {
```

Comment:
Another string_view by reference - check if it can be by value

Reply:
Can you put a commit on top here too.